dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -413,6 +413,27 @@
 			label = "PWM_7";
 		};
 
+		adc0: adc@400d1000 {
+			compatible = "nuvoton,npcx-adc";
+			#io-channel-cells = <1>;
+			reg = <0x400d1000 0x2000>;
+			interrupts = <10 0>;
+			clocks = <&pcc NPCX_CLOCK_BUS_APB1 NPCX_PWDWN_CTL4 4>;
+			pinctrl = <&alt6_adc0_sl   /* PIN45 */
```

Comment:
I think is worth a comment stating that the order of the pinctrl list is important and the code depends on the order being from ADC0 to ADC9 so it can appropriately enable only the ADC channel needed

Reply:
Done.